go.mod

```diff
@@ -18,7 +18,7 @@ require (
 	github.com/tendermint/tendermint v0.34.14
 	github.com/tendermint/tm-db v0.6.6
 	golang.org/x/net v0.0.0-20211015210444-4f30a5c0130f // indirect
-	golang.org/x/sys v0.0.0-20220412211240-33da011f77ad // indirect
+	golang.org/x/sys v0.0.0-20220422013727-9388b58f7150 // indirect
```

Comment:
Should we keep this outside this PR?

Reply:
ideally, yes, but we certainly don't have to if it's just the result of `go mod tidy` call

Reply:
At the risk of being anal, `go mod tidy` should be done in a separate PR.

Reply:
will revert then